tests/results.txt

```diff
@@ -3352,6 +3352,12 @@ char w       ' digit?        cycle_test 2drop      CYCLES:     86 ok
              ' dodowordij    cycle_test            CYCLES: 165748 ok
 : doword+loop 100 0 do 5 +loop ;  ok
              ' doword+loop   cycle_test            CYCLES:   2418 ok
+: doifword 0 100 0 do i 2 and if 1 else -1 then + loop drop ;  ok
+             ' doifword      cycle_test            CYCLES:  26180 ok
+: buword 100 begin 1- ?dup 0= until ;  ok
```

Comment:
These are fine here for now, while you are working on things that affect multiple branching words, but they probably should be moved later down to the word you are trying to test (like `until` or `while`, which currently have no cycle tests).  Things are in more-or-less alphabetical order by the xt_xxx name (eg. `;` is in the "semicolon" location).  You're testing more than one word here, so it's up to you which you file it under.